V1/Services/CredentialService.cs

```diff
@@ -85,7 +85,7 @@ public async Task<CredentialsContract> CreateCovidTest(string connectionId, Covi
                 };
 
                 credentials = await _agencyBroker.SendCredentials(credentialOffer);
-                StoreCoviIDCredentials(covidTestCredential, walletId);
+                //StoreCoviIDCredentials(covidTestCredential, walletId);
```

Comment:
I know it wouldn't happen in this PR, but I'm guessing we can just delete this whole service?
Am I correct in saying this is replaced by TestResultService?

In fact, I think we can also get rid of the WalletService. Now that we're keeping the wallet ourselves, I think that can just be managed with the WalletRepository.

Reply:
This method will be removed later - I'll cleanup at the very end.
The wallet service needs to stick around, we need a layer before interacting with the database